rmgpy/rmg/model.py

```diff
@@ -858,6 +859,13 @@ def apply_kinetics_to_reaction(self, reaction):
                 reaction.reverse = None
         reaction.kinetics = kinetics
 
+        if isinstance(kinetics, (SurfaceArrhenius, StickingCoefficient, SurfaceArrheniusBEP, StickingCoefficientBEP)):
+            if kinetics.coverage_dependence:
+                for species, values in kinetics.coverage_dependence.items():
+                    species_in_model = self.make_new_species(species)
```

Comment:
`make_new_species` returns Species and bool (True if already in model, False if not), so this should be species_in_model, is_new = self.make_new_species(species)

Reply:
not sure about adding a reactive species to model from cov dep dict that may or may not be in model

Reply:
I think this still code block here still needs amending